adaptdl/adaptdl/torch/data.py

```diff
@@ -355,6 +355,23 @@ def skipdone(self):
         else:
             return False
 
+    def to_tensorboard(self, writer, global_step, tag_prefix=""):
+        """
+        Output some useful metrics to TensorBoard.
+
+        Arguments:
+            writer (torch.utils.tensorboard.SummaryWriter): ``SummaryWriter``
+                object to output metrics to.
+            global_step (int): Global step value to record.
+            tag_prefix (str): Prefix added to each metric's tag.
+        """
+        writer.add_scalar(tag_prefix + "Total_Batch_Size",
```

Comment:
I think this should be joined together with a `/`, otherwise tag_prefix will need to end with a `/`, which doesnt seem too intuitive to me.

Reply:
Alternatively, adding a `/` if the tag doesnt end in a `/` already